client/components/jetpack-decorative-card/index.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
```

Comment:
Minor, but since you're importing this only for the types, you can change this to the following to save a bunch of unused imports.

```suggestion
import type { FC } from 'react';
```

Reply:
Ah, nice one! Thanks!